ui/app/pages/add-token/add-token.component.js

```diff
@@ -274,31 +261,22 @@ class AddToken extends Component {
     )
   }
 
-  renderSearchToken () {
-    const { tokenSelectorError, selectedTokens, searchResults } = this.state
-
-    return (
-      <div className="add-token__search-token">
-        <TokenSearch
-          onSearch={({ results = [] }) => this.setState({ searchResults: results })}
-          error={tokenSelectorError}
-        />
-        <div className="add-token__token-list">
-          <TokenList
-            results={searchResults}
-            selectedTokens={selectedTokens}
-            onToggleToken={(token) => this.handleToggleToken(token)}
-          />
-        </div>
-      </div>
-    )
-  }
-
   renderTabs () {
+    const { tokenSelectorError, tokensToSearch } = this.state
+
     return (
       <Tabs>
         <Tab name={this.context.t('search')}>
-          { this.renderSearchToken() }
+          <SearchableItemList
+            itemsToSearch={transformTokensForSearchList(tokensToSearch)}
+            itemSelectorError={tokenSelectorError}
+            onClickItem={(token) => this.handleToggleToken(token)}
+            Placeholder={TokenListPlaceholder}
+            className="add-token__search-token"
+            searchPlaceholderText={this.context.t('searchTokens')}
+            fuseSearchKeys={[{ name: 'name', weight: 0.499 }, { name: 'symbol', weight: 0.499 }, { name: 'address', weight: 0.002 }]}
```

Comment:
Were these changes intentional? The old keys used were as follows:

```suggestion
            fuseSearchKeys={[{ name: 'name', weight: 0.5 }, { name: 'symbol', weight: 0.5 }]}
```

Reply:
These changes were intentional. They make a small UX improvement that allows the user to copy and paste a token address into the list and find that token. However, now that you ask, I realize this concern should be separate from this PR, so I will remove it